ethbloom/Cargo.toml

```diff
@@ -10,9 +10,9 @@ repository = "https://github.com/paritytech/parity-common"
 
 [dependencies]
 tiny-keccak = "1.4"
-crunchy = { version = "0.2.1", default-features = false, features = ["limit_256"] }
+crunchy = { version = "0.2", default-features = false, features = ["limit_256"] }
```

Comment:
Not relevant to this PR, but I wonder what code we're unrolling to need 256 levels?

Reply:
I guess that depends on a user-defined uint type size. That way we support numbers up to 256 * 8 bytes.